pkg/auth/auth.go

```diff
@@ -40,8 +41,6 @@ const (
 )
 
 var (
-	log = capnslog.NewPackageLogger("github.com/openshift/console", "auth")
```

Comment:
Do we want a unique log for auth or just put it all in the same file?

Reply:
I don't know that we feel this is important at this point.  We don't have a huge backend, so no need to do package oriented logging.